yarn.lock

```diff
@@ -435,6 +453,11 @@
   resolved "https://registry.yarnpkg.com/@babel/helper-validator-identifier/-/helper-validator-identifier-7.19.1.tgz#7eea834cf32901ffdc1a7ee555e2f9c27e249ca2"
   integrity sha512-awrNfaMtnHUr653GgGEs++LlAvW6w+DcPrOliSMXWCKo597CwL5Acf/wWdNkf/tfEQE3mjkeD1YOVZOUV/od1w==
 
+"@babel/helper-validator-identifier@^7.22.20":
```

Comment:
Multiple compatible versions of `"@babel/helper-validator-identifier` can be combined.

Reply:
That's also what I thought, but it turns out not to be the case 😭 . I tried that on https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5309 and had to revert.